docs/source/conf.py

```diff
@@ -108,6 +108,12 @@
 html_title = "datashuttle"
 
 
+# Redirect the webpage to another URL
+# Sphinx will create the appropriate CNAME file in the build directory
+# https://www.sphinx-doc.org/en/master/usage/extensions/githubpages.html
+html_baseurl = "https://datashuttle.neuroinformatics.dev/"
```

Comment:
Can remove this section, it is already at the very bottom of the `conf.py`. However, the comment is useful, if you could move this to the existing `html_baseurl = "https://datashuttle.neuroinformatics.dev/"` that would be great.